src/aioswitcher/api/__init__.py

```diff
@@ -732,16 +804,20 @@ async def stop_shutter(self) -> SwitcherBaseResponse:
         response = await self._reader.read(1024)
         return SwitcherBaseResponse(response)
 
-    async def set_position(self, position: int = 0) -> SwitcherBaseResponse:
-        """Use for setting the shutter position of the Runner and Runner Mini devices.
+    async def set_position(
+        self, position: int = 0, index: int = 0
+    ) -> SwitcherBaseResponse:
+        """Use for setting the shutter position of the Runners devices.
 
         Args:
             position: the position to set the device to, default to 0.
+            index: which runner to set position, default to 0.
 
         Returns:
             An instance of ``SwitcherBaseResponse``.
 
         """
+        index_packet = get_shutter_index(self._device_type, index) + 1
```

Comment:
Please add a comment explaining the `+1` part.

Reply:
It is done.

Reply:
I'm sorry, I'm not too familiar with Switcher shutters, so this still makes no sense to me.
But if Shay is okay with this, I'm too.

@thecode Are we okay with this +1 and the following comment:
```
# We need to convert the selected circuit number to the actual place in the packet?
```
Do you think this makes sense?

Reply:
I don't fully understand the range of the valid indexes for a device since we don't have documentation for how each device works. 

I think we at least need to specify the valid range of this method and what does it mean (circuit number? what is the value for a single circuit device?)

As for the +1, If I understand correctly there are different indexes ranges used by the discovery packet and the API calls, if this is correct might be better to have two methods: `get_discovery_packet_index` and `get_api_packet_index`.